AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_Atom_LevelLoadTest.py

```diff
@@ -0,0 +1,85 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+def Atom_LevelLoadTest():
```

Comment:
what script is executing this hydra script? it should be in one of the `TestSuite_Main.py`, `TestSuite_Main_GPU.py`, or `TestSuite_Sandbox.py` scripts unless it's being called somewhere else

Reply:
Added to Main test suite, and ran it to make sure it passed, test result added to comments